### ISSUE TITLE
buildah-commit: return value is not image ID of image created

### ISSUE THREAD

**Issue by taqtiqa-mark** [projectatomic/buildah#609]
**Description**
From the documentation for [`buildah commit...`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-commit.md):
>RETURN VALUE
The image ID of the image that was created. On error, 1 is returned and errno is returned.

**Steps to reproduce the issue:**
1. OCI_ID=$(${BUILDAH} commit ${OCI_NAME} ${OCI_NAME})
2. echo $OCI_ID

**Describe the results you received:**
````bash
````

**Describe the results you expected:**
````bash
<some-signature or 1>
````

**Output of `apt list buildah`:**

```
buildah/xenial 0.16-1~ubuntu16.04.2~ppa4 amd64
```

**Output of `buildah version`:**

```
Version:       0.16
Go Version:    go1.8.7
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    f5a76e0
Built:         Fri Apr  6 15:33:05 2018
OS/Arch:       linux/amd64
```

**Output of `cat /etc/*release`:**

```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.4 LTS"
NAME="Ubuntu"
VERSION="16.04.4 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.4 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```

**Output of `uname -a`:**

```
Linux <redacted> 4.4.0-119-generic #143-Ubuntu SMP Mon Apr 2 16:08:24 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Output of `cat /etc/containers/storage.conf`:**

```
# storage.conf is the configuration file for all tools
# that share the containers/storage libraries
# See man 5 containers-storage.conf for more information

# The "container storage" table contains all of the server options.
[storage]

# Default Storage Driver
driver = "overlay"

# Temporary storage location
runroot = "/var/run/containers/storage"

# Primary read-write location of container storage
graphroot = "/var/lib/containers/storage"

[storage.options]
# AdditionalImageStores is used to pass paths to additional read-only image stores
# Must be comma separated list.
additionalimagestores = [
]

# Size is used to set a maximum size of the container image.  Only supported by
# certain container storage drivers (currently overlay, zfs, vfs, btrfs)
size = ""

# OverrideKernelCheck tells the driver to ignore kernel checks based on kernel version
override_kernel_check = "true"
```

**Comment by TomSweeneyRedHat** [comment]
@taqtiqa-mark I think (hope) this was fixed very recently with #566 which was merged just a few days ago.  If you get a chance can you get the absolute latest/greatest and retest please?

**Comment by taqtiqa-mark** [comment]
Thanks, it is fixed in the latest.
